crates/sui-core/src/scoring_decision.rs

```diff
@@ -7,13 +7,14 @@ use arc_swap::ArcSwap;
 use narwhal_types::ReputationScores;
 use std::collections::HashMap;
 use std::sync::Arc;
+use sui_simulator::anemo::PeerId;
 use sui_types::base_types::AuthorityName;
 use sui_types::committee::Committee;
-use tracing::{debug, info, warn};
+use tracing::{info, warn};
 
 // TODO: migrate these values to config
 const MAD_DIVISOR: f64 = 0.7;
-const CUTOFF_VALUE: f64 = 2.4;
+const CUTOFF_VALUE: f64 = 6.0;
```

Comment:
Just for my own understanding, by increasing this we will see less low scoring authorities?

Reply:
Yes, I think it was too sensitive based on the graphs